### PR TITLE
Add kutl tests for TLS deployments

### PR DESCRIPTION
**Pull request by vakwetu**

### CODE REVIEW

File: tests/kuttl/tests/dataplane-deploy-tls-test/00-dataplane-create.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands: |
+  oc deploy -f ./deploy/certs-create.yaml
```
Comment (jpodivin) [inline tests/kuttl/tests/dataplane-deploy-tls-test/00-dataplane-create.yaml:4]: I'm afraid you have to use `script` subfield for this.
https://kuttl.dev/docs/testing/steps.html#shell-scripts

**Review by jpodivin — CHANGES_REQUESTED** [review]
Fails while trying to convert yaml to struct.
`harness.go:394: loading /go/src/github.com/openstack-k8s-operators/dataplane-operator/tests/kuttl/tests/dataplane-deploy-tls-test/00-dataplane-create.yaml: error converting unstructured object TestStep:/ (/go/src/github.com/openstack-k8s-operators/dataplane-operator/tests/kuttl/tests/dataplane-deploy-tls-test/00-dataplane-create.yaml): error converting TestStep:/ from unstructured error: cannot restore slice from string`

**Comment by vakwetu** [comment]
Closing --

**Comment by vakwetu** [comment]
superseded by https://github.com/openstack-k8s-operators/dataplane-operator/pull/651

**Pull request closed without merging**